network/iptables.md

```diff
@@ -98,10 +98,11 @@ _default_, it does not _restrict_ services to that IP.
 
 If you are running Docker version 20.10.0 or higher with [firewalld](https://firewalld.org){: target="blank" rel="noopener" class=“”} on your system with `--iptables` enabled, Docker automatically creates a `firewalld` zone called `docker` and inserts all the network interfaces it creates (for example, `docker0`) into the `docker` zone to allow seamless networking.
 
-Consider running the following `firewalld` command to remove the docker interface from the zone.
+In case you have manually inserted a docker network interface to another zone, please remove the interface from that zone, using the commands below - 
 
 ```bash
 # Please substitute the appropriate zone and docker interface
+$ firewall-cmd --get-zone-of-interface=docker0
```

Comment:
This only shows the zone, or does this also remove it?

Perhaps we should split this to a separate paragraph; show steps to

1. "find" the current zone
2. remove the zone

Then, show how to restart docker (?)